common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1032,28 +1032,52 @@ public static String computeMaxHostBrokerProtocol() {
 
         /**
          * Signature info for Intune Company portal app that installs authenticator
-         * component.
+         * component. Generated with SHA-1.
          */
         public static final String COMPANY_PORTAL_APP_RELEASE_SIGNATURE = "1L4Z9FJCgn5c0VLhyAxC5O9LdlE=";
```

Comment:
Yeah, they're referenced in some redirect uri checks and tests. Though the brokerHost sha1 signature isn't currently being used, I don't think it would hurt to keep it until we actually change the redirect uri ourselves.